### DISCUSSION TITLE
Skin elements positioning and splitting

### DISCUSSION THREAD

**Discussion by Hexality** [ppy/osu#14048] [Ideas] [answered]
Hey, what's up?
Came to ask if in the future the elements on the skin editor will be able to be moved using the arrow keys for a precise positioning, will it be possible in the future?

also wanted to ask if in the future it will be possible to separate both the scorebar for its background for a fine tuning in elements, cause skins like mine that have a centered lifebar with the user icon in the middle always keep imperfectly connected because of the background.

Edit: detailing a bit, its basically the possibility to remove the scorebar-bg from one of the elements and keep only 1 of them (if you use 2 lifebars like I do to center like this) with it or completely removing it for an example.
![image](https://user-images.githubusercontent.com/17398632/127369623-86239afc-899c-42d1-bdfe-4dd30f5038fd.png)

**Comment by bdach** [comment]
Nudging will presumably be implemented, the same way the beatmap editor has it.

I wouldn't hold my breath on the scorebar stuff, though. The legacy skinning systems are purposefully not expanded any further than whatever capabilities stable had.

**Reply by Hexality** [reply]
yeah but for now the game has no way of adding custom images for skinning so idk

**Comment by peppy** [comment, marked as answer]
yes to both, eventually.
